### ISSUE TITLE
Is lit.min.js meant to exist?

### ISSUE THREAD

**Issue by jonnyarnold** [lit/lit#1961]
Hi there!

I'm trying to upgrade to lit v2.0.0-rc.2 from using `lit-html` only in v1. We use [browser module loading](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules), which doesn't play nicely with the import statements in `lit/index.js`:

```js
// My project's file, customElements.js. 
// Note that this import will be handled by the browser,
// and we host our JS at /static/**
import { LitElement, html, nothing } from "/static/lit/index.js"

// The first line of lit/index.js, which will fail because:
// 1. We serve files with a /static prefix
// 2. Browser-based imports require a file extension:
//    https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#importing_features_into_your_script
import "@lit/reactive-element"
```

Because lit-html was `import`-free in v1, we didn't have this issue. So I've been looking for a bundled version of lit v2 which resolves the imports.

It looks like there's a [rollup config to create a `lit.min.js` file](https://github.com/lit/lit/blob/main/packages/lit/rollup.config.js#L52); however, it doesn't exist in the package - I *think* because the [`files` in `package.json` does not allow it](https://github.com/lit/lit/blob/main/packages/lit/package.json#L74).

We've only had to use the TypeScript compiler up to now and it would be a shame to have to include a bundler just because lit doesn't have an `import`-free version.

To summarise, I have a couple of questions...

1. Is `lit.min.js` meant to be part of the package files?
2. Does an `import`-free version of lit v2 exist?

Thanks for any help you can give :)

**Comment by justinfagnani** [comment]
`lit-html` still doesn't import any other package by name in 2.x. If you were using it without a node module resolution step before it should still work.

`lit` is a combination of `lit-html` and `lit-element`, and it does import those with bare specifiers. If you want to use `lit` you need to have a node module resolution step. You can just use `lit-html` though.

We're going to publish a few different pre-bundled versions to GitHub or lit.dev for those who don't use npm. We won't really recommend it since it's an easy way to get package duplication or load a lot of code you don't need for unused directives and such.

**Comment by jonnyarnold** [comment]
Thanks for the comprehensive response @justinfagnani - I'll try getting our Django + TypeScript project to play nicely with the imports.
